test/server/routes/updateUser.js

```diff
@@ -12,15 +11,23 @@ test("testing 'api/v1/:userId' update route", (t) => {
     .then((id) => {
       supertest(app)
         .put(`/api/v1/users/${id}`)
+        .set('Cookie', [
+          'jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX25hbWUiOiJBbmdoYW0xMTYiLCJyb2xlIjoiRGV2ZWxvcGVyIiwiaWF0IjoxNTU2NTM5ODA2fQ.LH9KjeeekNZ0PAogdB8qr3Ew7V6sz_Ih9cjLSOpVNN4',
+        ])
```

Comment:
@aminalakhsham It might save you some time in the future if you take this big cookie value (the `jwt=eyJh....`) and put it into its own file somewhere (for example `/test/server/constants/testCookie.js`) and export it, and then import it into these files and use it.
Then, in the future if you ever had to change the cookie value for some reason, you could just change it in that one file, instead of having to change it in these ~15 files.